drivers/hmis/lib/form_data/assessments/dummy_intake_assessment.json

```diff
@@ -41,58 +41,71 @@
           "pick_list_reference": "LengthOfStay"
         },
         {
-          "type": "GROUP",
-          "link_id": "pls-conditionals",
-          "item": [
+          "type": "DISPLAY",
+          "link_id": "3.917.A",
+          "read_only": true,
+          "text": "Client stayed 90+ days in an institutional setting. This is considered a \"break\" according to the HUD definition of chronic homelessness. Stopping data collection for 3.917 Prior Living Situation.",
+          "project_types_excluded": ["ES", "SERVICES_ONLY", "SH"],
+          "enable_behavior": "ALL",
+          "enable_when": [
             {
-              "type": "BOOLEAN",
-              "link_id": "3.917.A",
-              "text": "Did the client stay less than 90 days?",
-              "project_types_excluded": ["ES", "SERVICES_ONLY", "SH"],
-              "enable_behavior": "ALL",
-              "enable_when": [
-                {
-                  "question": "3.917.1",
-                  "operator": "EQUAL",
-                  "answer_group_code": "INSTITUTIONAL"
-                }
-              ]
+              "question": "3.917.1",
+              "operator": "EQUAL",
+              "answer_group_code": "INSTITUTIONAL"
             },
             {
-              "type": "BOOLEAN",
-              "link_id": "3.917.B",
-              "text": "Did the client stay less than 7 nights?",
-              "project_types_excluded": ["ES", "SERVICES_ONLY", "SH"],
-              "enable_behavior": "ALL",
-              "enable_when": [
-                {
-                  "question": "3.917.1",
-                  "operator": "EQUAL",
-                  "answer_group_code": "TEMPORARY_PERMANENT_OTHER"
-                }
+              "question": "3.917.2",
+              "operator": "IN",
+              "answer_codes": [
+                "LOS_90_DAYS_OR_MORE_BUT_LESS_THAN_ONE_YEAR",
+                "LOS_ONE_YEAR_OR_LONGER"
               ]
+            }
+          ]
+        },
+        {
+          "type": "DISPLAY",
+          "link_id": "3.917.B",
+          "text": "Client stayed 7+ days in a transitional or permanent housing situation. This is considered a \"break\" according to the HUD definition of chronic homelessness. Stopping data collection for 3.917 Prior Living Situation.",
```

Comment:
This is now simply a conditionally displayed message. I think this pattern could be really useful.